web_tool_external_stress_test.R

```diff
@@ -118,8 +120,7 @@ if (exists("portfolio")) {
     dplyr::mutate(
       loss = exposure * shock / 100,
       sector = ifelse(is.na(sector), "Other", sector)
-    ) %>%
-    dplyr::ungroup()
+    )
```

Comment:
just to go sure, since I am not good with groupings and cannot test the code, you moved the grouping from l 123 (after the mutate) to l 112 (before the mutate), this has no impact no the results? @jacobvjk

Reply:
yes, we are not doing column operations here, so this ought to make no difference, not sure why it was part of the grouping in the first place